### ISSUE TITLE
Can we expect the other word-embeddings e.g. Word2Vec and GLOVE on the "indicnlp_corpus" soon?

### ISSUE THREAD

**Issue by skmalviya** [ai4bharat-indicnlp/indicnlp_corpus#3]

**Comment by anoopkunchukuttan** [comment]
Other word embeddings are not an immediate priority. Given the previous literature, FastText looks like a good choice for word embeddings and we are busy with other tasks currently. We have shared the text corpus, and you should be able to train these models on a CPU yourself.  That would be a nice contribution! We will be happy to link to your models if and when you share them.

**Comment by skmalviya** [comment]
Thanks for the prompt reply. I accept the suggestion and would like to contribute. I have read your paper and learn about the corpus as well. In the [indicnlp.v1.hi.vec.gz](https://storage.googleapis.com/ai4bharat-public-indic-nlp-corpora/embedding/indicnlp.v1.hi.vec.gz) vec file, there are **924720** word-embeddings but the [Hindi-corpus](https://storage.googleapis.com/ai4bharat-public-indic-nlp-corpora/data/monolingual/indicnlp_v1/sentence/hi.txt.gz) consists of **5322627** token-types in total. 

1. Does the following **fasttext** training step read the unprocessed data directly or it accepts the pre-processed data only 

```
$FASTTEXT_HOME/build/fasttext skipgram \
	-epoch 10 -thread 30 -ws 5 -neg 10    -minCount 5 -dim 300 \
	-input $mono_path \
	-output $output_emb_prefix 
```
2. Is there any pre-processing of the corpus needed to be done before the training? If yes then please suggest the preprocessing criteria used in the paper.

**Comment by anoopkunchukuttan** [comment]
- We have created word embedding for words with frequency>5, hence the difference from the type count.
- What we have shared with you is the pre-processed corpus (tokenization and normalization done). These were done using the IndicNLP library. So, no processing is needed from  your end.

**Comment by skmalviya** [comment]
Thanks for clarifying the doubt. I will let you know when it's done!
